app/workers/builds.php

```diff
@@ -181,6 +181,10 @@ protected function buildDeployment(Document $project, Document $function, Docume
             $build->setAttribute('stderr', $response['stderr']);
             $build->setAttribute('stdout', $response['response']);
 
+            /* Also update the deployment buildTime */
+            $deployment->setAttribute('buildTime', $response['duration']);
+            $deployment = $dbForProject->updateDocument('deployments', $deployment->getId(), $deployment);
```

Comment:
we don't need to update this right?

Reply:
I added this to help with events not receiving buildTime, if you don't mind about events getting it, we can remove it

Reply:
yeah, but do we need to update the document?